qemu/tests/cfg/fwcfg_enable.cfg

```diff
@@ -22,13 +25,23 @@
         windbg_path = "x86\windbg.exe"
     x86_64:
         windbg_path = "x64\windbg.exe"
-    Win8..1:
-        unzip_cmd = powershell.exe -nologo -noprofile -command "& { Add-Type -A 'System.IO.Compression.FileSystem'; [IO.Compression.ZipFile]::ExtractToDirectory('%s:\Memory.dmp.zip', '%s:'); }"
-        chk_id_cmd = 'type %s | find /i "Bugcheck code 00000161"'
     chk_windbg_cmd = 'dir "C:\Program Files (x86)\Windows Kits\10\Debuggers\%s"'
     feature = "OptionId.WindowsDesktopDebuggers"
-    windbg_install_cmd = "WIN_UTILS:\winsdksetup.exe /features %s /q"
-    chk_dump_cmd = "WIN_UTILS:\AutoIt3_%PROCESSOR_ARCHITECTURE%.exe WIN_UTILS:\check_dump_windbg.au3"
+    sdk_setup = winsdksetup.exe
+    Win8..1, Win2012..r1, Win2012..r2:
+        sdk_setup = sdksetup_81.exe
+        unzip_cmd = powershell.exe -nologo -noprofile -command "& { Add-Type -A 'System.IO.Compression.FileSystem'; [IO.Compression.ZipFile]::ExtractToDirectory('%s:\Memory.dmp.zip', '%s:'); }"
+        chk_windbg_cmd = 'dir "C:\Program Files (x86)\Windows Kits\8.1\Debuggers\%s"'
```

Comment:
Current implementation is okay for me. Just we have better way to get the NT kernel version for the path(8.1/10), also for the build version(1607/1809), we might improve it later.

Reply:
Got it, thanks~